components/MainButton.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
I am not a fan of these names:
- ExitGameButton
- MainButton
- SecondaryButton

For reasons and ways to improve, see https://v2.vuejs.org/v2/style-guide/?redirect=true#Order-of-words-in-component-names-strongly-recommended

Reply:
What about?

ButtonExitGame // button with specific purpose and a svg icon used only once in the app
ButtonMain  // generic text only button meant to be more prominent than the secondary button
ButtonSecondary

Reply:
Good 👍